app/assets/javascripts/components/jquery_wrappers/TaggedInput.js

```diff
@@ -1,6 +1,11 @@
 import React from 'react';
 
 class TaggedInput extends React.Component {
+  constructor() {
```

Comment:
You're not, but that's not the reason to pass the props up, your constructor doesn't use the props but the _parent constructor_ (the one of `React.Component`) does.